doc/runtimes/hugo.md

```diff
@@ -12,7 +12,7 @@ details on components and configuration of build images.
 
 The Hugo toolset is run when one of following conditions met:
 
-1. `config.yaml`, `config.toml`, `config.yml` or `config.json` configuration files in root of repo.
+1. `config.yaml`, `config.toml`, `config.yml`, `config.json` or `hugo.*` equivalent configuration files in root of repo.
```

Comment:
Could you please clarify why supporting "hugo.*"? Looking quickly at the doc I could only see "hugo.toml" being used. Am I missing something?

Reply:
That makes sense! Updating it right now